### ISSUE TITLE
Issues on the imitation data

### ISSUE THREAD

**Issue by bluehalk** [mlech26l/icra_lds#1]
Hello, I hope you are doing well, sir.

I have a few questions regarding the imitation data. I would greatly appreciate your assistance with them.

Firstly, there are 29 files in the .npz file. However, in your paper, you mentioned that you collected 20 training sets, 5 validation sets, and 5 test sets. I would like to clarify whether the 29 files in the .npz file include all the training and validation data, and if I should split it into train and validation datasets on my own. Also, is there no test data provided?

Secondly, I have noticed that when I treat the Lidar-collision problem as a CNN problem and use 32 as the number of channels instead of timesteps, the results are much better than those obtained with an RNN model. I'm having difficulty understanding this. Does it imply that the data itself is not well-suited for time-series modeling?

I look forward to your response. Thank you.

**Comment by mlech26l** [comment]
1. This seems to be a typo. There are 5 validation files and the remaining 24 files are for training (the 5:5:24 split was only used for HP tuning). There is no test set because the actual test is on the real robot.

2. You need to check whether your temporal CNN is anti-causal or not. If you treat the time axis as a convolution axis, the resulting model is anti-causal meaning that it has access to future observation, therefore cannot be deployed on the robot

**Comment by bluehalk** [comment]
> 1. This seems to be a typo. There are 5 validation files and the remaining 24 files are for training (the 5:5:24 split was only used for HP tuning). There is no test set because the actual test is on the real robot.
> 2. You need to check whether your temporal CNN is anti-causal or not. If you treat the time axis as a convolution axis, the resulting model is anti-causal meaning that it has access to future observation, therefore cannot be deployed on the robot

Thanks for your answering! It really helps. It seems that I made a logical mistake.
